geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/iterators/IteratorClassLoader.scala

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2013-2015 Commonwealth Computer Research, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Apache License, Version 2.0 which
+ * accompanies this distribution and is available at
+ * http://www.opensource.org/licenses/apache2.0.php.
+ */
+
+package org.locationtech.geomesa.accumulo.iterators
+
+import com.typesafe.scalalogging.slf4j.Logging
+import org.apache.commons.vfs2.impl.VFSClassLoader
+import org.geotools.factory.GeoTools
+
+object IteratorClassLoader extends Logging {
```

Comment:
I didn't update all the other iterators - so we should keep that ticket open. but this should make it a lot easier.